src/assets/index.ts

```diff
@@ -0,0 +1,11 @@
+import deviceImg from './images/device.jpg';
+import azureLogo from './images/microsoft-azure.jpg';
+import logo from './images/logo.jpg';
+import errorImg from './images/error.jpg';
+
+export default {
```

Comment:
It would be better to use PascalCase for images, like react components.
Please use postfix `Image` or `Img` for each of them

`deviceImg` → `DeviceImg / DeviceImage`